README.md

```diff
@@ -72,175 +103,206 @@ To access details of each command:
-and add the address to the `.env` file:
+<img src="static/generate-account-faucet.png" width="900px" alt="Get catalog"/>
 
-```.env
-ETH_ACCOUNT=<ETHEREUM_ACCOUNT_ADDRESS>
-```
+**Note:** _This account will be automatically funded by the Faucet service configured previously._
+
+> **Important:** _Set `FROM_ACCOUNT` value in `.env` to the address created (it will be used to send transactions)_
 
-> We don't reuse the same address as the faucet address to showcase that the newly created account will be funded automatically.
+If using Rinkeby you can verify that the account has been properly pre-funded by the Faucet using Etherscan (following url: https://rinkeby.etherscan.io/address/<ACCOUNT_ADDRESS> after replacing placeholder)
 
-### Consuming transaction messages
+<img src="static/etherscan-account.png" width="900px" alt="Get catalog"/>
 
-_PegaSys is a transaction orchestrator. Transactions on Ethereum being asynchronous, you will receive the result of a transaction by consuming messages coming from Orchestrate._
+### Consume transaction receipts
 
-To do so, open two tabs on your terminal. In one tab we will consume messages and check the transaction receipt. On the other, we will send transactions. On the first tab, run:
+> **Note:** _Orchestrate allows to manage blockchain transactions that are by nature asynchronous due to blockchain mining time. Orchestrate SDK provides a consumer that allows to asynchronously process transaction receipts._
+
+> **Note:** _Orchestrate uses Apache Kafka for asynchronous communications_
+
+On a first terminal window start the consumer:
 
 ```bash
 npm run consume
 ```
 
-### Deploying a contract
+<img src="static/npm-consume.png" width="900px" alt="Get catalog"/>
+
+> **Reference script:** [consume.ts](src/consume/consume.ts)
+
+> **Important**: _Leave this terminal window open without interupting consumer_
 
-We will deploy a contract on the Rinkeby testnet. On the second tab, run:
+### Deploy a Smart Contract
 
-> Have a look at the script `src/deploy-contract/deploy.ts` for reference
+> **Important:** _Before moving forward make sure that `FROM_ACCOUNT` in `.env` has been set_
+
+On a second terminal window, run deployment script:
 
 ```bash
 npm run deploy
 ```
 
-Verify in the first tab that you received the message successfully and copy-paste the value of the `contractAddress` in the `.env` file:
+> **Reference script:** [deploy.ts](src/deploy-contract/deploy.ts)
 
-```.env
-COUNTER_CONTRACT_ADDRESS=<COUNTER_CONTRACT_ADDRESS>
-```
+After a few seconds (depending on blocktime) you should see the transaction receipt appearing in the consumer terminal
 
-### Sending a transaction
+<img src="static/npm-deploy.png" width="900px" alt="Get catalog"/>
 
-We will now send a transaction to the `Counter` contract we just deployed to increment the counter by 1.
+> **Important:** _Set `TO_ACCOUNT` value in `.env` with the address of the contract just deployed (you can find it in the receipt at `contractAddress`)_
```

Comment:
make it a required step not juste an important note. Use notes only for side information that can be important to know but not something you have to do right now for this QS.
Indicate where to find the address in the JSON